{"uuid":"6697d6c0-4f63-467d-af24-7bc770195803","historyId":"2c1c91fa31092cea4a48f13df6738c1a","fullName":"died_proceso_nombramiento.nomInscripcion.object_test.NomLoginTest.logInToTheSessionCaseTest01","labels":[{"name":"package","value":"died_proceso_nombramiento.nomInscripcion.object_test.NomLoginTest"},{"name":"testClass","value":"died_proceso_nombramiento.nomInscripcion.object_test.NomLoginTest"},{"name":"testMethod","value":"logInToTheSessionCaseTest01"},{"name":"parentSuite","value":"Login"},{"name":"suite","value":"Test_02"},{"name":"subSuite","value":"died_proceso_nombramiento.nomInscripcion.object_test.NomLoginTest"},{"name":"host","value":"ALQ000002139"},{"name":"thread","value":"6972@ALQ000002139.main(1)"},{"name":"epic","value":"Regression Tests"},{"name":"feature","value":"Login Tests"}],"links":[],"name":"logInToTheSessionCaseTest01","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Usuario error] but found [Debe llenar este campo.]","trace":"java.lang.AssertionError: expected [Usuario error] but found [Debe llenar este campo.]\r\n\tat org.testng.Assert.fail(Assert.java:93)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:512)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:134)\r\n\tat org.testng.Assert.assertEquals(Assert.java:115)\r\n\tat org.testng.Assert.assertEquals(Assert.java:189)\r\n\tat org.testng.Assert.assertEquals(Assert.java:199)\r\n\tat died_proceso_nombramiento.nomInscripcion.objectpages.NomLoginPage.verifyLoginUserName_aroundBody2(NomLoginPage.java:101)\r\n\tat died_proceso_nombramiento.nomInscripcion.objectpages.NomLoginPage$AjcClosure3.run(NomLoginPage.java:1)\r\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\r\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\r\n\tat died_proceso_nombramiento.nomInscripcion.objectpages.NomLoginPage.verifyLoginUserName(NomLoginPage.java:101)\r\n\tat died_proceso_nombramiento.nomInscripcion.object_test.NomLoginTest.logInToTheSessionCaseTest01(NomLoginTest.java:57)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:661)\r\n\tat org.testng.internal.Invoker.retryFailed(Invoker.java:1014)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:744)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:380)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1226)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1144)\r\n\tat org.testng.TestNG.run(TestNG.java:1115)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","start":1549553139895,"stop":1549553144081,"steps":[{"name":"Open N11 Step...","status":"passed","stage":"finished","start":1549553139896,"stop":1549553141419},{"name":"Login Step with username: {0}, password: {1}, for method: {method} step...","status":"passed","stage":"finished","start":1549553141921,"stop":1549553142178,"parameters":[{"name":"row","value":"<xml-fragment r=\"2\" spans=\"1:23\" x14ac:dyDescent=\"0.25\" xmlns:mc=\"http://schemas.openxmlformats.org/markup-compatibility/2006\" xmlns:r=\"http://schemas.openxmlformats.org/officeDocument/2006/relationships\" xmlns:x14ac=\"http://schemas.microsoft.com/office/spreadsheetml/2009/9/ac\" xmlns:main=\"http://schemas.openxmlformats.org/spreadsheetml/2006/main\">\r\n  <main:c r=\"A2\" s=\"10\" t=\"s\">\r\n    <main:v>19</main:v>\r\n  </main:c>\r\n  <main:c r=\"B2\" s=\"10\" t=\"s\">\r\n    <main:v>0</main:v>\r\n  </main:c>\r\n  <main:c r=\"C2\" s=\"10\" t=\"s\">\r\n    <main:v>1</main:v>\r\n  </main:c>\r\n  <main:c r=\"D2\" s=\"10\" t=\"s\">\r\n    <main:v>2</main:v>\r\n  </main:c>\r\n  <main:c r=\"E2\" s=\"10\" t=\"s\">\r\n    <main:v>3</main:v>\r\n  </main:c>\r\n  <main:c r=\"F2\" s=\"10\" t=\"s\">\r\n    <main:v>4</main:v>\r\n  </main:c>\r\n  <main:c r=\"G2\" s=\"10\" t=\"s\">\r\n    <main:v>14</main:v>\r\n  </main:c>\r\n  <main:c r=\"H2\" s=\"10\" t=\"s\">\r\n    <main:v>15</main:v>\r\n  </main:c>\r\n  <main:c r=\"I2\" s=\"10\" t=\"s\">\r\n    <main:v>26</main:v>\r\n  </main:c>\r\n</xml-fragment>"}]},{"name":"Verify username: {0} step...","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Usuario error] but found [Debe llenar este campo.]","trace":"java.lang.AssertionError: expected [Usuario error] but found [Debe llenar este campo.]\r\n\tat org.testng.Assert.fail(Assert.java:93)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:512)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:134)\r\n\tat org.testng.Assert.assertEquals(Assert.java:115)\r\n\tat org.testng.Assert.assertEquals(Assert.java:189)\r\n\tat org.testng.Assert.assertEquals(Assert.java:199)\r\n\tat died_proceso_nombramiento.nomInscripcion.objectpages.NomLoginPage.verifyLoginUserName_aroundBody2(NomLoginPage.java:101)\r\n\tat died_proceso_nombramiento.nomInscripcion.objectpages.NomLoginPage$AjcClosure3.run(NomLoginPage.java:1)\r\n\tat org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149)\r\n\tat io.qameta.allure.aspects.StepsAspects.step(StepsAspects.java:46)\r\n\tat died_proceso_nombramiento.nomInscripcion.objectpages.NomLoginPage.verifyLoginUserName(NomLoginPage.java:101)\r\n\tat died_proceso_nombramiento.nomInscripcion.object_test.NomLoginTest.logInToTheSessionCaseTest01(NomLoginTest.java:57)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)\r\n\tat java.base/java.lang.reflect.Method.invoke(Unknown Source)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:661)\r\n\tat org.testng.internal.Invoker.retryFailed(Invoker.java:1014)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:744)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:380)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1226)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1144)\r\n\tat org.testng.TestNG.run(TestNG.java:1115)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","start":1549553143179,"stop":1549553143238,"parameters":[{"name":"expectedUserNameMessage","value":"Usuario error"}]}],"attachments":[{"name":"Page screenshot","source":"c7b9f077-5d09-4092-a84b-7e5a86d24ad1-attachment","type":"image/png"},{"name":"{0}","source":"82c0f0ca-f7b8-4b20-985a-505f320fd4cf-attachment","type":"text/plain"}],"parameters":[]}